js/toolbar/annotation-tool.js

```diff
@@ -65,17 +68,27 @@ function AnnotationTool(parent){
                     "<input type='color' value='" + _self.curStroke + "' id='strokeColor'/>",
                 "</span>",
                 "<span class='toolBtn' data-type='PATH'>",
```

Comment:
The same way that I added `title=` to the two last options to explain what they are, please add tooltips for the other options as well.

Reply:
made the changes